src/components/picker/picker.tsx

```diff
@@ -318,12 +323,18 @@ export class Picker {
     }
 
     private createChip(listItem: ListItem): Chip {
+        const name = getIconName(listItem.icon);
+        const backgroundColor = getIconBackgroundColor(
+            listItem.icon,
+            listItem.iconColor
+        );
+
         return {
             id: `${listItem.value}`,
             text: listItem.text,
             removable: true,
-            icon: listItem.icon,
-            iconBackgroundColor: listItem.iconColor,
+            icon: name,
+            iconBackgroundColor: backgroundColor,
```

Comment:
something is wrong here. the chosen chip's color is not applied
<img width="391" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/339b8f18-651c-4c72-b102-f19cd8ca00e0">

also tested this: 
```tsx
return {
            id: `${listItem.value}`,
            text: listItem.text,
            removable: true,
            icon: {
                name: name,
                backgroundColor: backgroundColor,
            },
            value: listItem,
        };
``` 
but got the same result